skimage/util/colormap.py

```diff
@@ -258,4 +259,266 @@
+              [0.989363, 0.557873, 0.391671],
+              [0.990138, 0.565296, 0.395122],
+              [0.990871, 0.572706, 0.398714],
+              [0.991558, 0.580107, 0.402441],
+              [0.992196, 0.587502, 0.406299],
+              [0.992785, 0.594891, 0.410283],
+              [0.993326, 0.602275, 0.414390],
+              [0.993834, 0.609644, 0.418613],
+              [0.994309, 0.616999, 0.422950],
+              [0.994738, 0.624350, 0.427397],
+              [0.995122, 0.631696, 0.431951],
+              [0.995480, 0.639027, 0.436607],
+              [0.995810, 0.646344, 0.441361],
+              [0.996096, 0.653659, 0.446213],
+              [0.996341, 0.660969, 0.451160],
+              [0.996580, 0.668256, 0.456192],
+              [0.996775, 0.675541, 0.461314],
+              [0.996925, 0.682828, 0.466526],
+              [0.997077, 0.690088, 0.471811],
+              [0.997186, 0.697349, 0.477182],
+              [0.997254, 0.704611, 0.482635],
+              [0.997325, 0.711848, 0.488154],
+              [0.997351, 0.719089, 0.493755],
+              [0.997351, 0.726324, 0.499428],
+              [0.997341, 0.733545, 0.505167],
+              [0.997285, 0.740772, 0.510983],
+              [0.997228, 0.747981, 0.516859],
+              [0.997138, 0.755190, 0.522806],
+              [0.997019, 0.762398, 0.528821],
+              [0.996898, 0.769591, 0.534892],
+              [0.996727, 0.776795, 0.541039],
+              [0.996571, 0.783977, 0.547233],
+              [0.996369, 0.791167, 0.553499],
+              [0.996162, 0.798348, 0.559820],
+              [0.995932, 0.805527, 0.566202],
+              [0.995680, 0.812706, 0.572645],
+              [0.995424, 0.819875, 0.579140],
+              [0.995131, 0.827052, 0.585701],
+              [0.994851, 0.834213, 0.592307],
+              [0.994524, 0.841387, 0.598983],
+              [0.994222, 0.848540, 0.605696],
+              [0.993866, 0.855711, 0.612482],
+              [0.993545, 0.862859, 0.619299],
+              [0.993170, 0.870024, 0.626189],
+              [0.992831, 0.877168, 0.633109],
+              [0.992440, 0.884330, 0.640099],
+              [0.992089, 0.891470, 0.647116],
+              [0.991688, 0.898627, 0.654202],
+              [0.991332, 0.905763, 0.661309],
+              [0.990930, 0.912915, 0.668481],
+              [0.990570, 0.920049, 0.675675],
+              [0.990175, 0.927196, 0.682926],
+              [0.989815, 0.934329, 0.690198],
+              [0.989434, 0.941470, 0.697519],
+              [0.989077, 0.948604, 0.704863],
+              [0.988717, 0.955742, 0.712242],
+              [0.988367, 0.962878, 0.719649],
+              [0.988033, 0.970012, 0.727077],
+              [0.987691, 0.977154, 0.734536],
+              [0.987387, 0.984288, 0.742002],
+              [0.987053, 0.991438, 0.749504]]
+
+
 viridis = LinearSegmentedColormap.from_list('viridis', viridis_data)
+cm.register_cmap('viridis', viridis)
```

Comment:
I would do this only if they are not already registered, so as not to overwrite whatever ships with matplotlib >1.5, which might have some improvements over what we ship here, for example.